### ISSUE TITLE
can i build in xcode13？

### ISSUE THREAD

**Issue by xzk49** [giginet/NESEmulator-watchOS#6]

**Comment by adityajs12321** [comment]
@13825046403 Yes, just make sure to change the swift compiler version to swift 5 in build settings

**Comment by xzk49** [comment]
Do you have a compiled project or ipa?



发自我的iPhone


------------------ Original ------------------
From: adityajs12321 ***@***.***&gt;
Date: Sat,Apr 2,2022 10:30 PM
To: giginet/NESEmulator-watchOS ***@***.***&gt;
Cc: xzk ***@***.***&gt;, Mention ***@***.***&gt;
Subject: Re: [giginet/NESEmulator-watchOS] can i build in xcode13？ (Issue #6)





 
@13825046403 Yes, just make sure to change the swift compiler version to swift 5 in build settings

**Comment by adityajs12321** [comment]
I recently made a fork with updated project code (https://github.com/adityajs12321/NESEmulator-watchOS8), can't send a .ipa file though as that may cause more errors than it solves. Just follow the build instructions given in that

**Comment by giginet** [comment]
Sorry, this repository is no longer maintained. 
It may need to migrate xcodeproj.
Forks and PRs are welcome. 😄

**Comment by EnderRobber101** [comment]
> I recently made a fork with updated project code (https://github.com/adityajs12321/NESEmulator-watchOS8), can't send a .ipa file though as that may cause more errors than it solves. Just follow the build instructions given in that

Hey adityajs12321 can you tell me how to update to swift 4 or 5 please. I am new to xcode
